src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1307,11 +1310,17 @@ private async Task ValidateUnusedIdsAsync(HashSet<string> unusedIds, string data
 
         }
 
-        public static unsafe bool IsBase64String(string base64)
+        public static unsafe bool IsBase64Id(string base64)
```

Comment:
should be called IsDatabaseId? since this is not base64 actually?

Reply:
Yes, I changed it